Security/src/EOMT.ps1

```diff
@@ -794,6 +794,10 @@ try {
             $RegMessage = "Server is vulnerable"
             Set-LogActivity -Stage $Stage -RegMessage $RegMessage -Message $Message
             Run-Mitigate
+
+            $Message = "EOMT has mitigated against known attacks. For long-term protection, please use Microsoft Update to install the latest Security Update for Exchange Server (KB5000871)."
```

Comment:
If the CU which the Exchange server is on does not have the SU, the admins will not be seeing any updates in Windows update, how are we planning to handle that scenario?

Reply:
Wouldn't the Get-ServerVulnStatus conditioning this block catch that? It's looking to see if the user is on a newer CU before it bothers to check whether they've installed the hotfix.

Reply:
I am not talking about the newer CU, this is regarding the CUs which security patches have not been released